landingpage/src/components/AlterConfigForName.tsx

```diff
@@ -34,9 +40,18 @@ const AlterConfigForName: React.FC<{
 		setAlterKeyInput(value === 'any' ? '' : undefined);
 	};
 
-	const RAW_ASCII_ScriptValue = `${queryString}.${selectedOption === 'any' ? alterKeyInput : selectedOption}=${alterValueInput}`;
-	const opReturnScript =
-		`script(OP_RETURN ${util.asciiToHex(RAW_ASCII_ScriptValue)})` as const;
+	useEffect(() => {
+		const RAW_ASCII_ScriptValue = `${queryString}.${selectedOption === 'any' ? alterKeyInput : selectedOption}=${alterValueInput}`;
+		setInscription(RAW_ASCII_ScriptValue)
+	}, [alterKeyInput, alterValueInput, selectedOption]);
```

Comment:
IDE complains that `queryString` should be added to the array, but I know that it never changes in this case